rewrite-weblogic/documentation/content/Migrate to WebLogic 141100.md

```diff
@@ -0,0 +1,89 @@
+# Migrate to WebLogic 14.1.1
+**com.oracle.weblogic.rewrite.UpgradeTo1411**
+
+These recipes apply the changes required for upgrading to WebLogic Server 14.1.1.0.0 and for upgrading from Java 8 to Java 11.
+
+### Tags:
+  - weblogic
+  - java
+
+### Recipe Source
+
+[weblogic-14.1.1.yaml](https://github.com/oracle/rewrite-recipes/blob/main/rewrite-weblogic/src/main/resources/META-INF/rewrite/weblogic-14.1.1.yaml)
+
+### Recipe List:
+  - [org.openrewrite.java.migrate.Java8toJava11](https://docs.openrewrite.org/recipes/java/migrate/java8tojava11)
+  - com.oracle.weblogic.rewrite.UpgradeTo1411
+
+### Usage
+
+These recipes will update the WebLogic version to 14.1.2 for the Maven build and will apply changes commonly needed when upgrading to Java 11. Specifically, for those applications that are built on Java 8, the recipe will update and add dependencies on J2EE libraries that are no longer directly bundled with the JDK and will also replace deprecated API with equivalents when there is a clear migration strategy.
+
+#### Maven POM
+
+1. Add the following to your Maven POM file.
+```
+<plugin>
+    <groupId>org.openrewrite.maven</groupId>
+    <artifactId>rewrite-maven-plugin</artifactId>
+    <version>5.43.0</version>
+    <configuration>
+        <exportDatatables>true</exportDatatables>
+        <activeRecipes>
+            <recipe>com.oracle.weblogic.rewrite.UpgradeTo1411</recipe>
+            <recipe>org.openrewrite.java.migrate.Java8toJava11</recipe>
+        </activeRecipes>
+    </configuration>
+    <dependencies>
+        <dependency>
+            <groupId>com.oracle.weblogic.rewrite</groupId>
+            <artifactId>rewrite-weblogic</artifactId>
+            <version>0.4.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.openrewrite.recipe</groupId>
+            <artifactId>rewrite-migrate-java</artifactId>
+            <version>2.28.0</version>
+        </dependency>
+    </dependencies>
+</plugin>
+...
+```
+2. Run `mvn rewrite:run` to run the recipe.
+
+#### Gradle
+
+1. Add the following to your `build.gradle` file:
+
+```
+plugins {
+    id("org.openrewrite.rewrite") version("7.1.4")
+}
+
+rewrite {
+    activeRecipe("com.oracle.weblogic.rewrite.UpgradeTo1411")
+    activeRecipe("org.openrewrite.java.migrate.Java8toJava11")
+    setExportDatatables(true)
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    rewrite("org.openrewrite.recipe:rewrite-migrate-java:3.2.0")

```

Comment:
Is missing to add a line for `rewrite("com.oracle.weblogic.rewrite:rewrite-weblogic:0.4.2")`